lib/drivers/medtronic/medtronicSimulator.js

```diff
@@ -68,7 +68,9 @@ exports.make = function(config){
   };
 
   function setSuspendingEvent(event) {
-    suspendingEvent = event;
```

Comment:
Same here, just wasn't sure if I should revert these changes on this branch, since Gerrit merged them in from master.